setup.py

```diff
@@ -23,7 +23,7 @@
     version=version,
     packages=find_packages(exclude=['contrib', 'docs', 'tests*']),
     include_package_data=True,
-    install_requires=['requests', 'pytz', 'json-stream==1.3.0'],
+    install_requires=['requests', 'pytz', 'json-stream==1.3.0','psutil'],
```

Comment:
hi, thanks a lot, nice to see you here!

we install psutil as part of the apiproxy extras, basically I wanted to keep the basic package as light as possible in terms of dependencies and then add what's needed using extras because I assume most users won't need the additional features and just want to use the api via HTTP